packages/edit-post/src/editor.js

```diff
@@ -92,7 +92,10 @@ function Editor( {
 				hiddenBlockTypes: getHiddenBlockTypes(),
 				blockTypes: getBlockTypes(),
 				template:
-					supportsTemplateMode && isViewable && canEditTemplate
+					supportsTemplateMode &&
+					isViewable &&
+					canEditTemplate &&
+					currentPost.postType === 'post'
```

Comment:
Not sure this will work, what about custom post types?

Reply:
Ah, true, had overlooked custom post types. Have switched to check that it isn't `wp_template` instead.